src/sentry/api/endpoints/organization_events.py

```diff
@@ -249,7 +249,7 @@ def get(self, request: Request, organization) -> Response:
             )
 
         on_demand_metrics_enabled = (
-            batch_features.get("organizations:on-demand-metrics-extraction", False)
+            batch_features.get("organizations:on-demand-metrics-extraction-widgets", False)
```

Comment:
Do we want to exclusively let people use this endpoint for widgets? I assume for now it's being only used by widgets but I can see it being needed in the future for other on demand queries. I would keep the `or` condition like in the `events-stats`, wdyt?

Reply:
I will add it back 👍🏻